t/lib/OpenQA/SeleniumTest.pm

```diff
@@ -380,17 +381,19 @@ sub wait_for_element {
     return $element;
 }
 
+sub kill_webapi() {
+    $webapi->signal('TERM');
+    $webapi->finish;
+}
+
 sub kill_driver() {
     return unless $startingpid && $$ == $startingpid;
     if ($_driver) {
         $_driver->quit();
         $_driver->shutdown_binary;
         $_driver = undef;
     }
-    if ($webapi) {
-        $webapi->signal('TERM');
-        $webapi->finish;
-    }
+    kill_webapi;
```

Comment:
This will trigger ugly warnings if the variable $webapi is undefined

Reply:
The code was checking that before, but we never unset it as far as I can tell.

Reply:
But do we call `kill_driver` without setting it?

Reply:
It's also unset is the process was stopped in before. And there can be cases where the webapi has been stopped and then kill_driver is called, e.g. when tests fail but should still shut down orderly